### PR TITLE
Add title tags to pages

### PR DESCRIPTION
**Pull request by stefannibrasil**
Space Station card: https://trello.com/c/dMpjhelx/734-seo-improvements-ubersuggest-add-title-tag-to-https-designthoughtbotcom-sprint-guide?

We want to have titles for our pages for SEO improvements.

I just copied the ones from https://github.com/thoughtbot/design-dot

### CODE REVIEW

File: eleventy.config.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ module.exports = function(eleventyConfig) {
   // Usage: {% breadcrumbs page.url %}
   eleventyConfig.addShortcode("breadcrumbs", function(url) {
     const parts = url.split("/").filter((part) => part.length)
-    const links = ["<a href='/'>Home</a>"]
+    const links = ["<a href='/sprint-guide'>Home</a>"]
```
Comment (stefannibrasil) [inline eleventy.config.js:42]: The breadcrumbs was broken for me. It seems the redirect set here is not working:

https://github.com/thoughtbot/design-sprint-guide/blob/main/netlify.toml#L7

Or is that config only set for production?

File: sprint-guide/exercises/schedule-interviews.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
-tags: ['exercise', 'planning']
+tags: ["exercise", "planning"]
 title: Schedule 4–6 people for testing interviews
 ---
 
-During the [Test phase](/phases/test), we'll be running user interviews and
+During the [Test phase](/sprint-guide/phases/test), we'll be running user interviews and
```
Comment (stefannibrasil) [inline sprint-guide/exercises/schedule-interviews.md:6]: Same as [above.](https://github.com/thoughtbot/design-sprint-guide/pull/113/files#r1571516395)

File: sprint-guide/schedules/schedules.njk [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-     pagination: {
+    pagination: {
         data: "sprint_guide.schedules",
         size: 1,
         alias: "schedule",
         },
     permalink: "/sprint-guide/schedules/{{ pagination.items[0] | slug }}/",
-    layout: "layouts/sprint-guide/base"
+    layout: "layouts/sprint-guide/base",
+    title: "Schedules"
```
Comment (stefannibrasil) [inline sprint-guide/schedules/schedules.njk:10]: I tried to get evaluate this to be the schedule title but could not make it work: `{{ sprint_guide.schedules[schedule].title }}`

This block seem to only accept strings for the title?

**Review by garciadanny — APPROVED** [review]

File: eleventy.config.js (hunk 1 above)
Comment (garciadanny) [reply eleventy.config.js:42]: I don't know enough about this to be able to answer this question or this [one](https://github.com/thoughtbot/design-sprint-guide/pull/113/files#r1571518389). Other than that, I reviewed your changes and didn't see anything that stood out to me. LGTM 👍

Comment (stefannibrasil) [reply eleventy.config.js:42]: Thank you, Danny! I will merge this as it is, then. These links were not working for me locally.

File: eleventy.config.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
 const { EleventyRenderPlugin } = require("@11ty/eleventy");
 const markdown = require("markdown-it")({
-  html: true
-})
+  html: true,
```
Comment (stefannibrasil) [inline eleventy.config.js:3]: I reverted a change and my linter went crazy on this file xD

File: sprint-guide/exercises/schedule-interviews.md (hunk 2 above)
Comment (stefannibrasil) [reply sprint-guide/exercises/schedule-interviews.md:6]: This one is currently broken, so will keep this change.

**Pull request merged**